.java-version

```diff
@@ -0,0 +1 @@
+1.8
```

Comment:
Nothing specific to `AggregateFactory`, but ut is a good idea to tell people what JVM version are we using to avoid problems with JDK specific changes. This file comes from jenv and if you us it it will switch local JVM to version 1.8. In times of JDK 8, 9, 10, 11, 12, 13 out there, it is a good thing we learned in projects. But you are completely right, it has nothing to do with this PR. I'll move it to a dedicated PR.